_data/projects/nodatime.yml

```diff
@@ -0,0 +1,12 @@
+name: Noda Time
+desc: A better date/time API for .NET
+site: https://github.com/nodatime/nodatime
+tags:
+- C#
```

Comment:
We're mostly figuring out our tag set over in #678 but this is worth doing for the moment.